scripts/recordid-to-filepath.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
Is such a script necessary? 

I'm using a simple one-liner to achieve the same needs:

```console
$ rg '^    "recid": "123"' data/records
• data/records/cms-hlt-configuration-files-2010.json
732:    "recid": "123",
```

Note also the leading space formatting in the regexp, which helps to avoid false positives, because record ID can be referenced in links. 

(This technique works with less false positives, because all the fixture files are formatted in the same way, using `scripts/clean_json_file.py`.)

Reply:
As far as I've seen, `Ripgrep` is not installed on the shipped web container by default (where following the opendata development guide, all commands for setup and working with records are executed).

With the records I have tested, my script should also avoid false positives (order by leading whitespaces). 

I think the command is quite important for developing and fixing issues with certain records, so if not as a separate script, this or yours should be at least added to the dev documentation somewhere.

Reply:
If `rg` is not available, you could also use `git grep` or `grep -r`.

(Personally I'm always grepping locally, i.e. outside of the container.)